UI/JustAssembly/Nodes/AssemblyNode.cs

```diff
@@ -10,11 +9,15 @@
 using JustAssembly.Interfaces;
 using JustAssembly.MergeUtilities;
 using JustAssembly.Nodes.APIDiff;
+using JustAssembly.Dialogs.DangerousResource;
```

Comment:
The ordering of the using should be - framework usings, libraries usings and internal usings.